Vtulkin/bot.py

```diff
@@ -0,0 +1,178 @@
+		И загружать из файла при запуске.
+
+		Получение актуальной информации из интернета.
+	"""
+
+	@staticmethod
+	def find_out_condition(condition):
+		""" Перевести описание погоды для метода find_out_weather"""
+		d = {"clear": "ясно", "partly-cloudy": "малооблачно", "cloudy": "облачно с прояснениями",
+			"overcast": "пасмурно", "partly-cloudy-and-light-rain": "небольшой дождь", "partly-cloudy-and-rain": "дождь",
+			"overcast-and-rain": "сильный дождь",  "overcast-thunderstorms-with-rain":
+			"сильный дождь, гроза", "cloudy-and-light-rain":
+			"небольшой дождь", "overcast-and-light-rain":
+			"небольшой дождь","cloudy-and-rain":
+			"дождь", "overcast-and-wet-snow":
+			"дождь со снегом","partly-cloudy-and-light-snow":
+			"небольшой снег","partly-cloudy-and-snow":
+			"снег","overcast-and-snow":
+			"снегопад","cloudy-and-light-snow":"небольшой снег",
+			"overcast-and-light-snow": "небольшой снег","cloudy-and-snow": "снег"
+			}
+		for key in d.keys():
+			if condition == key:
+				return d[key]
+		return "Непонятные условия погоды."
+
+	def find_out_weather(self) -> str:
+		""" Запрос погоды через Yandex API"""
+		api_key = "ae76725b-fa0a-4b4e-a41b-38e5e8d03037"
+		# широта и долгота нужного города (для Читы "52.033973" и "113.499432")
+		lat = "52.033973"
+		lon = "113.499432"
+		try:
+			headers = {'X-Yandex-API-Key': api_key}
+			res = requests.get("https://api.weather.yandex.ru/v1/informers?lat={}&lon={}".format(lat, lon), headers=headers)
+			# print(res.status_code)
+			data = res.json()
+			stroka = self.find_out_condition(str(data["fact"]['condition']))
+			answer = "Температура (°C): " + str(data["fact"]["temp"]) + \
+					 "\nОщущается как: " + str(data["fact"]['feels_like']) + \
+					 "\n" + stroka + \
+					 "\nВетер: " + str(data["fact"]['wind_speed']) + " м/c"
+			return answer
+		except Exception as e:
+			return "Что-то пошло не так." + e
+
+	def get_history_of_chatting(self) -> dict:
+		return self._rec
+
+	def write_to_json(self):
+		""" Запись в файл"""
+		with open("users/"+self._userName+".json", "w") as file:
+			json.dump(self._rec, file, indent="")
+
+	def read_from_json(self):
+		""" Чтение из файла с именем пользователя"""
+		try:
+			with open("users/"+self._userName+".json", "r") as file:
+				self._rec = json.load(file)
+		except FileNotFoundError:
+			self._rec = {}
+
+	def fill_dict(self, key, data):
+		""" ЗАполнить запись"""
+		self._rec[key] = data
+
+	def set_name_of_user(self, name):
+		""" задает имя пользователя и ищет подходящий файл с историей сообщений"""
+		if isinstance(name, str):
+			self._userName = name
+			self.read_from_json()
+
+	def __init__(self, user_name="user"):
+		""" Конструктор для инициализации имени пользователя и истории сообщений"""
+		self._userName = user_name  # Имя пользователя
+		self._rec = {}
+		self.read_from_json()  # инициализируется свойство _rec для хранения истории переписки в виде словаря
+
+	def reply(self, s) -> str:
```

Comment:
в python можно указывать желаемый тип данных и для параметров

```suggestion
	def reply(self, s:str) -> str:
```